packages/ui/src/core/tamagui.ts

```diff
@@ -16,6 +16,13 @@ export {
   TextArea,
   Text,
   Dialog,
+  Separator,
```

Comment:
Is this used? Would prefer to remove it.

Reply:
Removed